alt_e2eshark/onnx_tests/models/nlp.py

```diff
@@ -28,6 +34,23 @@ def __init__(self, *args, **kwargs):
         def update_dim_param_dict(self):
             self.dim_param_dict = dim_param_dict
 
+        def construct_inputs(self):
+            """Overrides the parent class method to construct sample inputs with the correct dimensions."""
+            default_inputs = super().construct_inputs()
+
+            tensors = list(default_inputs.data)
+
+            self.update_sess_options()
+            session = ort.InferenceSession(self.model, self.sess_options)
```

Comment:
I don't really want to load an inference session again for this. Can we store a list of corresponding input names when running construct inputs the first time (maybe in the test info class)? This might be useful for other purposes, too.

Reply:
Let's merge this anyway for now.